twitter_analysis/README.md

```diff
@@ -4,7 +4,7 @@ TweetMind is a project that analyzes public issues and gives a biased public ove
 ## How to run the application?
 
   1. Build docker image by running the following command within the docker file directory. docker build --tag python-docker .
-  2. Run this command to run the docker image: docker run -p 5000:5000 -t -i python-docker:latest
+  2. Run this command to run the docker image: docker run -p 5000:5000 -t -i -v "$(pwd)/folder_1:/app/folder_1" python-docker:latest
```

Comment:
Do you create this folder named "folder_1"? How do you know this folder exists?

Reply:
It's created on the go in the docker image as well as the local folder.